packages/website/src/components/ChatConfiguration.tsx

```diff
@@ -188,38 +189,6 @@ export const ThemeEditor: FC<{
   );
 };
 
-const RadioList = <T extends unknown>({
```

Comment:
extracted to its own component. Peter wanted me to use this component for voice compass examples